print-pkg-names.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+import sys
+
+from packaging import utils
+
+
+def debug(msg: str):
+    print(f'::debug::{msg.title()}', file=sys.stderr)
+
+
+def safe_parse_pkg_name(file_path: pathlib.Path) -> str | None:
+    if file_path.suffix == '.whl':
+        try:
+            return utils.parse_wheel_filename(file_path.name)[0]
+        except utils.InvalidWheelFilename:
+            debug(f'Invalid wheel filename: {file_path.name}')
+            return None
+    elif file_path.suffix == '.gz':
+        try:
+            return utils.parse_sdist_filename(file_path.name)[0]
+        except utils.InvalidSdistFilename:
+            debug(f'Invalid sdist filename: {file_path.name}')
+            return None
+    return None
+
+
+packages_dir = pathlib.Path(sys.argv[1]).resolve().absolute()
```

Comment:
absolute does nothing here
```suggestion
packages_dir = pathlib.Path(sys.argv[1]).resolve()
```

Reply:
fixed (and opened https://github.com/pypa/gh-action-pypi-publish/pull/258 )

Reply:
Thanks! Marking this thread as resolved.